vdb/recon/__init__.py

```diff
@@ -85,8 +100,8 @@ def notify(self, event, trace):
         stackptr = trace.getStackCounter()
 
         rawargs = trace.readMemoryFormat(stackptr, '<%dP' % (len(self._reconfmt) + 1))
-        savedeip = rawargs[0]
-        args = rawargs[1:]
+        savedeip = trace.getProgramCounter()
+        args = getArgs(trace, len(self._reconfmt))
 
```

Comment:
perfect.  heading down this road, we'll hopefully manage to support windows arm without too much trouble :)